ideascube/forms.py

```diff
@@ -52,3 +59,29 @@ def save(self, *args, **kwargs):
         user.is_staff = True
         user.save()
         return user
+
+
+class UserImportForm(forms.Form):
+    source = forms.FileField(required=True)
+
+    def save(self):
+        source = self.cleaned_data['source']
+        users = []
+        errors = []
+        source = to_unicode(source.read().decode())
```

Comment:
bah, I was certainly tired yesterday night :p
Anyway, I'm waiting for your branch to be merged so I can use the Automagic thing :)